### DISCUSSION TITLE
Bug - API DELETE container from ghcr.io broken

### DISCUSSION THREAD

**Discussion by kc1r74p** [community/community#25688] [Actions]
<p>Hello, not sure when it broke, but I was using without an issue:<br>
<a href="https://docs.github.com/en/rest/reference/packages#delete-a-package-version-for-the-authenticated-user" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/rest/reference/packages#delete-a-package-version-for-the-authenticated-user</a></p>
<p>For example my versions look like this:</p>
<pre><code class="lang-auto">[
    {
        "id": 144,
        "name": "sha256:c65058fa7519270.............",
        ....
        "metadata": {
            "package_type": "container",
            "container": {
                "tags": [
                    "latest"
                ]
            }
        }
    },
    {
        "id": 123,
        "name": "sha256:d3c4f........",
        .....
        "metadata": {
            "package_type": "container",
            "container": {
                "tags": []
            }
        }
    }
]
</code></pre>
<p>And when I try to delete the old untagged version:</p>
<pre><code class="lang-auto">DELETE /user/packages/container/xxxx/versions/123
</code></pre>
<p>I get the error:</p>
<pre><code class="lang-auto">{
    "message": "You cannot delete the last tagged version of a package. You must delete the package instead.",
    "documentation_url": "https://docs.github.com/rest/reference/packages#delete-a-package-version-for-the-authenticated-user"
}
</code></pre>
<p>Obviously I do not try to delete the last tagged version.<br>
Also the same error seems to occur if I randomize the ID to a non existing.<br>
^ Improved error message would be nice.</p>
<p>Can someone tell me if this is a Bug, or did I miss some change?<br>
I know it worked a few weeks/months ago.<br>
Manually using the UI still works just fine but the API seems broken.</p>
<p>Best regards<br>
Patrick</p>

**Comment by gdomingu** [comment, marked as answer]
<p>Thank you very much for reporting this bug. I have pushed up a fix for it. Please let me know if there are any further issues.</p>

**Comment by kc1r74p** [comment]
<p>Works again :tada:<br>
Also thanks for improving the output message.</p>
